repository/rabbitmq/operator/params.yaml

```diff
@@ -25,4 +25,3 @@ parameters:
   - name: DISK_SIZE
     description: "Disk size for the RabbitMQ servers"
     default: "1Gi"
-    trigger: "not-allowed"
```

Comment:
how come you can drop this? 🤔

Reply:
There is no plan `not-allowed`. This is creating an error when running `kubectl kudo package verify`.

Reply:
oh I see, so I think the proper fix then is to create a plan with this task https://github.com/kudobuilder/operators/pull/283/files#diff-03f33d8320fe5aa10c3409ce4fc7189bL37

Reply:
Ah, so this then triggers a plan that does nothing instead of `deploy`/`update`, right? How did that ever work before?